dev/io.openliberty.pages.4.0.internal_fat/test-applications/Pages40Changes.war/resources/errorPage.jsp

```diff
@@ -0,0 +1,22 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+ <html>
+    <title>errorPage</title>
+    <body>
+        <%@ page isErrorPage="true"  %>
+        <% out.println("This is a test error page"); %>
+        <% out.println("Exception encountered is: " + exception.getMessage()); %>
+        <br/>
+        <!-- Verifying Pages 4.0 queryString:  -->
+        queryString: ${pageContext.errorData.queryString}
+
+         <!-- TODO VERIFY METHOD  -->
```

Comment:
Add to follow on issue

Reply:
https://github.com/OpenLiberty/open-liberty/issues/27128